migrations/versions/0229_new_letter_rates.py

```diff
@@ -0,0 +1,52 @@
+"""empty message
+
+Revision ID: 0229_new_letter_rates
+Revises: 0228_notification_postage
+
+"""
+
+revision = '0229_new_letter_rates'
+down_revision = '0228_notification_postage'
+
+import uuid
+from datetime import datetime
+from alembic import op
+
+
+start = datetime(2018, 9, 30, 23, 0)
+
+NEW_RATES = [
+    (uuid.uuid4(), start, 1, 0.30, False, 'second'),
```

Comment:
We don't need new rate for `second` class, let's remove these please. By the way if you were to insert new rates for second we would need to expire the old second class rates, it doesn't make sense to have to valid rates.

Reply:
Ok, I will do that :)